doc/contributing/technical-priorities.md

```diff
@@ -160,3 +160,18 @@ and integration with other systems.
 As TypeScript usage continues to grow and gains more prominence in the
 ecosystem, enhancing its support is essential for delivering an improved
 developer experience for newcomers and experienced users alike.
+
+## Package management
+
+The ability to easily install and manage dependencies and development tools is a
```

Comment:
```suggestion
The ability to easily install and manage dependencies and development tools is a
key part of the user experience, and for that reason Node.js must provide a
package manager as part of its distribution. Node.js includes `npm` for this
purpose and currently has no intentions to change this. This is for historical reasons
— when `npm` was added in 2011, it was
the only JavaScript package manager — and because it is the reference
implementation for the npm registry, which is the de facto primary source for
most JavaScript software.
```